### PR TITLE
Fix broken build due to *_nested dependencies

### PR DESCRIPTION
**Pull request by lorenzwalthert**
Actually I did not properly take into account the dependencies linked to *_nested() functions in #39 so the build broke. Hence, here comes the update. Since we are now using the visitor approach, I also updated the vignette.

@krlmlr Should we use `devtools::load_all()` at the beginning of the vignette? It seems like with the visitor concept, using `styler:::` everywhere adds too much noise everywhere.

### CODE REVIEW

**Comment by lorenzwalthert** [comment]
`pkgload::load_all()` seems to work yes.

**Comment by lorenzwalthert** [comment]
travis builds seem to work, but there is something [missing](https://ci.appveyor.com/project/lorenzwalthert/styler-bp2vm/build/1.0.75) related to pkgload to get a successful built with appveyor. I could not quite figure out what is missing in the appveyor file. Maybe you @krlmlr can help?

File: DESCRIPTION [hunk 1]
```diff
@@ -12,15 +12,17 @@ Imports:
     tibble,
     tidyr,
     purrr,
-    magrittr
+    magrittr,
+    pkgload
```
Comment (krlmlr) [inline DESCRIPTION]: I prefer alphabetic ordering of dependencies. Don't need to change anything now, just keep in mind for the next time we edit here.

**Comment by krlmlr** [comment]
I registered styler with my AppVeyor account, we should change the README badge at some point. Rtools seem to be installed, running a build to see if the error is reproducible.

**Pull request merged**